components/DrawControls.svelte

```diff
@@ -64,6 +81,7 @@
   onMount(async () => {
     drawControls = new MapboxDraw({
       displayControlsDefault: false,
+      userProperties: true,
```

Comment:
This was subtle; we don't get `intervention_type` or other props without this! And `user_` gets prefixed